ruma-events-macros/src/event_enum.rs

```diff
@@ -164,12 +265,14 @@ fn expand_any_enum_with_deserialize(
 ) -> syn::Result<TokenStream> {
     let attrs = &input.attrs;
     let event_type_str = &input.events;
-    let event_struct = Ident::new(&ident.to_string().trim_start_matches("Any"), ident.span());
+    let event_struct = Ident::new(&ident.to_string().replace("Any", ""), ident.span());
```

Comment:
This is becoming kind of brittle. With every single `event_enum!` generating more than the specified enum after this PR, could you rewrite it to parse predefined values, sth. like `kind: state` / `kind: message` instead of the current `name: AnyStateEvent` / `name: AnyMessageEvent`, in a followup PR?

Reply:
Yup I'll do that next.